### PR TITLE
Skins: improve color warnings

### PR DESCRIPTION
**Pull request by ronso0**
* make waveform range color warnings more useful
* comment out searchbox color info

### CODE REVIEW

File: src/waveform/renderers/waveformmarkrange.cpp [hunk 1]
```diff
@@ -43,6 +27,11 @@ WaveformMarkRange::WaveformMarkRange(
         DEBUG_ASSERT(!m_markEndPointControl); // has not been created yet
         m_markEndPointControl = std::make_unique<ControlProxy>(group, endControl);
     }
+    QString rangeName = startControl;
+    if (startControl.contains(QStringLiteral("_start_position"))) {
```
Comment (daschuer) [inline src/waveform/renderers/waveformmarkrange.cpp:31]: this parses the string twice.

Comment (daschuer) [reply src/waveform/renderers/waveformmarkrange.cpp:31]: This is only used in the error case, so I would duplicate it and move it closer to usage.
This way we are sure it is only executed when really needed.

**Review by daschuer — CHANGES_REQUESTED** [review]
Thank you. Some minor comments.

File: src/waveform/renderers/waveformmarkrange.cpp (hunk 1 above)
Comment (ronso0) [reply src/waveform/renderers/waveformmarkrange.cpp]: :+1:  Done.

File: src/waveform/renderers/waveformmarkrange.cpp [hunk 2]
```diff
@@ -61,6 +46,37 @@ WaveformMarkRange::WaveformMarkRange(
...
+
+    if (!m_disabledColor.isValid()) {
+        if (enabledControl.isEmpty()) {
+            m_disabledColor = QColor(Qt::transparent);
+        } else {
+            // Show warning only when there's no EnabledControl,
+            // like for intro & outro ranges.
+            QString rangeName = startControl;
+            if (startControl.contains(QStringLiteral("_start_position"))) {
+                rangeName = startControl.remove(QStringLiteral("_start_position"));
```
Comment (daschuer) [inline src/waveform/renderers/waveformmarkrange.cpp:72]: you can just call 
rangeName = startControl.remove(QStringLiteral("_start_position"));
the contais() check is done implicite and the if statement above redundant.

**Review by daschuer — CHANGES_REQUESTED** [review]

File: src/waveform/renderers/waveformmarkrange.cpp (hunk 2 above)
Comment (ronso0) [reply src/waveform/renderers/waveformmarkrange.cpp]: working on it..

**Comment by ronso0** [comment]
@daschuer it's compact now. Please check.

**Review by daschuer — APPROVED** [review]
Nice, LGTM. 
Waiting for CI

**Comment by daschuer** [comment]
The CI fails are unrelated. Thank you.

**Pull request merged**